config/owasp/suppressions.xml

```diff
@@ -17,20 +17,6 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: tomcat-embed-core-9.0.38.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
-        <cve>CVE-2020-13943</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.38.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cve>CVE-2020-13943</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
    file name: launchdarkly-java-server-sdk-5.1.1.jar (shaded: com.google.guava:guava:28.2-jre)
```

Comment:
update launch darkly to 5.2.2 and no need in suppression. Please do in all PR you raised